src/Utilities/WrapText.hpp

```diff
@@ -0,0 +1,15 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <string>
+
+/// \ingroup UtilitiesGroup
+/// \brief Wrap the string `str` so that it is no longer than `line_length` and
+/// indent each new line with `indentation`.
+///
+/// Single words longer than `line_length` are hyphenated.
```

Comment:
Document that the first line is not indented.  (Or maybe change it to indent the line, since both use cases indent that line themselves.)

Reply:
Good idea! I did this and pushed fixups as well as rebased on develop